zerver/management/commands/soft_activate_deactivate_users.py

```diff
@@ -42,18 +42,27 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         # type: (*Any, **str) -> None
-        realm = self.get_realm(options)
+        if settings.STAGING:
+            print('This is a Staging server. Suppressing management command.')
+            sys.exit(0)
```

Comment:
I'd do a blank line after this, for readability.  

Probably a few other places in this function would benefit from similar treatment, separating section of the code.